modules/migration-prerequisites.adoc

```diff
@@ -13,37 +13,31 @@ endif::[]
 * You must have `cluster-admin` privileges on all clusters.
 * The source and target clusters must have unrestricted network access to the replication repository.
 * The cluster on which the Migration controller is installed must have unrestricted access to the other clusters.
-ifdef::migrating-3-4,migrating-4-1-4,migrating-4-2-4[]
 * If your application uses images from the `openshift` namespace, the required versions of the images must be present on the target cluster.
 +
 If the required images are not present, you must update the `imagestreamtags` references to use an available version that is compatible with your application. If the `imagestreamtags` cannot be updated, you can manually upload equivalent images to the application namespaces and update the applications to reference them.
-endif::[]
-ifdef::migrating-3-4,migrating-4-2-4[]
-+
-The following `imagestreamtags` have been _removed_ from {product-title} 4.2:
 
-** `dotnet:1.0`, `dotnet:1.1`, `dotnet:2.0`
-** `dotnet-runtime:2.0`
-** `mariadb:10.1`
-** `mongodb:2.4`, `mongodb:2.6`
-** `mysql:5.5`, `mysql:5.6`
-** `nginx:1.8`
-** `nodejs:0.10`, `nodejs:4`, `nodejs:6`
-** `perl:5.16`, `perl:5.20`
-** `php:5.5`, `php:5.6`
-** `postgresql:9.2`, `postgresql:9.4`, `postgresql:9.5`
-** `python:3.3`, `python:3.4`
-** `ruby:2.0`, `ruby:2.2`
-endif::[]
-ifdef::migrating-3-4,migrating-4-1-4,migrating-4-2-4[]
-+
-The following `imagestreamtags` have been _removed_ from {product-title} 4.4:
+The following `imagestreamtags` have been _removed_ from {product-title} _4.2_:
 
-** `dotnet: 2.2`
-** `dotnet-runtime: 2.2`
-** `nginx: 1.12`
-** `nodejs: 8, 8-RHOAR, 10-SCL`
-** `perl:5.24`
-** `php: 7.0, 7.1`
-** `redis: 3.2`
-endif::[]
+* `dotnet:1.0`, `dotnet:1.1`, `dotnet:2.0`
+* `dotnet-runtime:2.0`
+* `mariadb:10.1`
+* `mongodb:2.4`, `mongodb:2.6`
+* `mysql:5.5`, `mysql:5.6`
+* `nginx:1.8`
+* `nodejs:0.10`, `nodejs:4`, `nodejs:6`
+* `perl:5.16`, `perl:5.20`
+* `php:5.5`, `php:5.6`
+* `postgresql:9.2`, `postgresql:9.4`, `postgresql:9.5`
+* `python:3.3`, `python:3.4`
+* `ruby:2.0`, `ruby:2.2`
+
+The following `imagestreamtags` have been _removed_ from {product-title} _4.4_:
```

Comment:
The way this is now isn't going to work for cherry picking this back to 4.3 and 4.2 (shouldn't be any references to 4.4 in 4.3 or 4.2).

However, I just checked the live 4.2 and 4.3 versions and do see this. So I'm fine with leaving this in this PR then, if you can do a followup PR to fix 4.2 and 4.3 to remove those 4.4 references. Are you okay with that?

Reply:
Actually, there was a reason for including the unsupported 4.4 image tags for the documentation for 4.2 and 4.3. The reason is that the user can, in theory, use a 4.2 CAM cluster to migrate from 3.7 to 4.4. Although the CAM cluster is normally also the target cluster, it does not have to be. 

OTOH, maybe that does complicate things too much. Since our default installation is for CAM to be installed on CAM, I will remove it from 4.2 and 4.3 in separate PR.